collections/hub/provisioning_dfsp/newdfsp.json

```diff
@@ -1091,6 +1091,154 @@
               }
             ]
           }
+        },
+        {
+          "id": 23,
+          "description": "Add callback - FX TRANSFERS POST",
+          "meta": {
+            "info": "Add DFSP callback - PARTIES SUB-ID ERROR PUT"
```

Comment:
Does this "info" field have proper value?

Reply:
Thanks for caching this. Copy paste issue, fixed it now.